packages/metrics/scripts/reportGenerator.ts

```diff
@@ -0,0 +1,859 @@
+        birthRow.birthState = fullComposition.eventState ?? ''
+        birthRow.birthCity = fullComposition.eventCity ?? ''
+        birthRow.healthCenter = fullComposition.healthCenter ?? ''
+        birthRow.officeLocation = fullComposition.officeLocation ?? ''
+
+        //Child details
+        birthRow.childGen = fullComposition.child?.gender ?? ''
+        birthRow.childDOB = fullComposition.child?.birthDate ?? ''
+        birthRow.childOrd = fullComposition.child?.multipleBirth ?? 0
+
+        //Mother details
+        birthRow.motherDOB = fullComposition.mother?.birthDate ?? ''
+        birthRow.motherMaritalStatus =
+          fullComposition.mother?.maritalStatus ?? ''
+        birthRow.motherOccupation = fullComposition.mother?.occupation ?? ''
+        birthRow.motherEducationalAttainment =
+          fullComposition.mother?.educational_attainment ?? ''
+        birthRow.motherCity = fullComposition.mother?.city ?? ''
+        birthRow.motherDistrict = fullComposition.mother?.district ?? ''
+        birthRow.motherState = fullComposition.mother?.state ?? ''
+
+        //Father details
+        birthRow.fatherDOB = fullComposition.father?.birthDate ?? ''
+        birthRow.fatherMaritalStatus =
+          fullComposition.father?.maritalStatus ?? ''
+        birthRow.fatherOccupation = fullComposition.father?.occupation ?? ''
+        birthRow.fatherEducationalAttainment =
+          fullComposition.father?.educational_attainment ?? ''
+        birthRow.fatherCity = fullComposition.father?.city ?? ''
+        birthRow.fatherDistrict = fullComposition.father?.district ?? ''
+        birthRow.fatherState = fullComposition.father?.state ?? ''
+
+        //Informant details
+        birthRow.informantDOB = fullComposition.informant?.birthDate ?? ''
+        birthRow.informantMaritalStatus =
+          fullComposition.informant?.maritalStatus ?? ''
+        birthRow.informantOccupation =
+          fullComposition.informant?.occupation ?? ''
+        birthRow.informantEducationalAttainment =
+          fullComposition.informant?.educational_attainment ?? ''
+        birthRow.informantCity = fullComposition.informant?.city ?? ''
+        birthRow.informantDistrict = fullComposition.informant?.district ?? ''
+        birthRow.informantState = fullComposition.informant?.state ?? ''
+        birthRow.informantRelationship =
+          fullComposition.informant?.relationship ?? ''
+
+        //Observations
+        birthRow.birthPluralityOfPregnancy =
+          fullComposition.observations?.birthPluralityOfPregnancy ?? ''
+        birthRow.bodyWeightMeasured =
+          fullComposition.observations?.bodyWeightMeasured ?? ''
+        birthRow.birthAttendantTitle =
+          fullComposition.observations?.birthAttendantTitle ?? ''
+        birthRow.presentAtBirthReg =
+          fullComposition.observations?.presentAtBirthReg ?? ''
+
+        // tslint:disable-next-line
+        console.log('Writing CSV row for birth declaration...')
```

Comment:
@sadmananik lets remove all the console logs from this function, instead I have a different idea regarding logging see notes below.